README.md

```diff
@@ -185,4 +196,3 @@ If you run your project's social media and would like the Foundation to share or
 
 For any other topics which aren't covered above, please email
 [operations@openjsf.org](mailto:operations@openjsf.org).
-
```

Comment:
Is there no longer a newline at the end of the document?

Reply:
Looks like there is still one.